api/app/signals/apps/services/domain/dsl.py

```diff
@@ -56,14 +56,22 @@ def __call__(self, signal: Signal):
             'areas': self.areas
         }
 
+        # add additonal question answers id to context
+        if signal.extra_properties:
+            for prop in signal.extra_properties:
+                if 'id' in prop and 'answer' in prop and prop['id'] not in tmp:
+                    tmp[prop['id']] = prop['answer']
+
+        return tmp
+
 
 class SignalDslService(DslService):
     context_func = SignalContext()
     signal_manager = SignalManager()
 
     def process_routing_rules(self, signal):
         ctx = self.context_func(signal)
-        rules = RoutingExpression.objects.select_related('_expression', '_department')
```

Comment:
We should manually add the standard Django ModelManager as an `objects` property to the  `signal.RoutingExpression` models. This would match the way the model managers are set up for the `signals.Signal` model.

Context: the idea was to have a manager that is more at the SIA business logic (`actions`) and one that is available if the business rules get in the way (the standard `objects` manager).

Reply:
check.